docs/docs/dynamo-store/access-strategies.md

```diff
@@ -70,3 +71,63 @@ With this in place Equinox will maintain and store a Snapshot event in the tip
 document that is guaranteed to always be up to date, and as a result the state
 of the Stream can be reconstructed with a single point-read.
 
+## RollingState(toSnapshot)
+
+`RollingState` will throw away events, only storing the snapshot. This can be
+useful for cases where we do not care about the individual events that led up to
+a state. This is frequently used to build up a read model that spans streams.
```

Comment:
```suggestion
a state. Such a mechanism is useful for storing derived data built from things that you do have a log of events for, e.g. an summary that indexes data from across multiple streams. The main thing to bear in mind is that the total (compressed) size of the state is limited to the DynamoDB Item size limit of 400K.
```

Reply:
maybe mention that it does concurrency control as normal, and you're able to test it end to end without docker or a shared table?